Misha/Files/mp3_tagger/tagger_tests.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+import eyed3
+from tagger_main import input_mp3_file_name, set_tag, get_tag
+
+
+class TaggerTest(unittest.TestCase):
+
+    def test_input_mp3_file_name_with_wrong_path(self):
+        with self.assertRaises(TypeError) as context:
+            self.assertRaises(TypeError, input_mp3_file_name('path/to/file/'))
+        exception_message = str(context.exception)
+        self.assertEqual('Invalid file type', exception_message)
+
+    def test_input_mp3_file_name_with_wrong_filename(self):
+        with self.assertRaises(OSError) as context:
+            self.assertRaises(OSError, input_mp3_file_name('file.mp3'))
+        exception_message = str(context.exception)
+        self.assertEqual('file not found', exception_message)
+
+    def test_tag_setter_with_wrong_dict(self):
+        audiofile = eyed3.load('Распродажа_на_AliExpress_Максим_Галкин.mp3')
+        tags_dict = {'artist': 'artist_name',
+                     'album': 'album_name',
+                     'album artist': 'alb_artist_name',
+                     'title': 'audio_title',
+                     'track number': (1, 1)}
+        with self.assertRaises(KeyError) as context:
+            self.assertRaises(KeyError, set_tag(audiofile, tags_dict))
+        exception_message = str(context.exception)
+        self.assertEqual('', exception_message)
+
+    def test_tag_setter_with_right_dict(self):
+        audiofile = eyed3.load('Распродажа_на_AliExpress_Максим_Галкин.mp3')
+        tags_dict = {'artist': 'artist_name',
+                     'album': 'album_name',
+                     'album artist': 'alb_artist_name',
+                     'title': 'audio_title',
+                     'track num': (1, 1)}
+        set_tag(audiofile, tags_dict)
```

Comment:
set tag or set tags? Function naming
Please add more tests for set_tag function

Reply:
SET TAG is a right name (built-in name is audiofile.TAG.title etc.)

i have added test for set_tag with big string like 'author'*1488